react/ConditionLayoutProduct.tsx

```diff
@@ -32,6 +33,7 @@ type SubjectArgs = {
   selectedItemId: { id: string }
   specificationProperties: { name: string; value?: string }
   areAllVariationsSelected: undefined
+  sellers: { isProductAvailable: boolean }
```

Comment:
What do you think of doing exactly like the `areAllVariationsSelected`?

We could replace `sellers` here with `isProductAvailable` and just return the ` isProductAvailable.length > 0`. The dev can control if they want it to be `false` or `true` via the `toBe` prop of the condition layout block.

Reply:
It's good idea. Thanks, @kaisermann . I'll try it.

Reply:
But...this way...will the product availability be verified to all of its sellers?

Reply:
@kaisermann I tried to implement your suggestion. Could you check it, pls?